modules/nextflow/src/util.py

```diff
@@ -64,3 +64,41 @@ def nextflow(
         result.get()
 
     return result, uuid
+
+def plot_fusion_only_arriba(
+    samples: Samples,
+    config: cfg.Config,
+    executor: executors.Executor,
+    workdir: Path,
+    # env: dict[str, str] | None = None,
+    # nxf_config: Path | None = None,
+    # nxf_work: Path | None = None,
+    # nxf_profile: str | None = None,
+    # ansi_log: bool = False,
+    # resume: bool = False,
+    name: str = "plot_arriba_only",
+    check: bool = True,
+    **kwargs
+) -> tuple[AsyncResult, UUID]:
+
+    uuid = uuid4() # generate random id
+
+    result, uuid = executor.submit(
+        str(Path(__file__).parent / "run_arriba.sh"),
+            f"-B {workdir}/arriba:/output", # need to refer to this output somewhere?
+            "-B /workspace/carolina/qd_rna/references:/references:ro",
```

Comment:
The location of the Arriba references will be somewhere under the path specified in `config.rnafusion.genomes_base`.